<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmUser</class>
 <widget class="QMainWindow" name="FrmUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form User</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>USER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="idUserLabel">
       <property name="text">
        <string>Id User</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Txt_id_user"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="namaLabel">
       <property name="text">
        <string>Nama</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Txt_nama"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Txt_password"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="roleLabel">
       <property name="text">
        <string>Role</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="Cmb_role">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Admin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="PB_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="PB_update">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="PB_exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="PB_del">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="PB_pesan">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
